larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -358,9 +366,9 @@ TfLiteStatus Prepare(KernelType kernel_type, const int bitwidth,
   return kTfLiteOk;
 }
 
-template <KernelType kernel_type, int bitwidth>
+template <KernelType kernel_type, const int default_bitpacking_bitwidth>
```

Comment:
I've never seen `const` in a template argument. Does it actually do anything?
```suggestion
template <KernelType kernel_type, int default_bitpacking_bitwidth>
```

Reply:
It seems to prevent assigning to the value in the body of the function - I tend to add const to everything reflexively and just remove it when I need to, because I'm too used to JavaScript and Scala and Rust :joy: I have removed it though now.